squad/core/models.py

```diff
@@ -1114,38 +1113,38 @@ def __init__(self, build, environment=None):
         self.tests_fail = 0
         self.tests_xfail = 0
         self.tests_skip = 0
-        self.failures = OrderedDict()
+        self.failures = defaultdict(list)
```

Comment:
the use of `OrderedDict` is usually to ensure consistent ordering of items in the UI, both web and email

Reply:
Oh, missed that one